airflow/www/views.py

```diff
@@ -71,8 +72,17 @@
 
 
 PAGE_SIZE = conf.getint('webserver', 'page_size')
+
+try:
+    async_dagbag_loader = conf.getboolean('webserver', 'async_dagbag_loader')
```

Comment:
```suggestion
async_dagbag_loader = conf.getboolean('webserver', 'async_dagbag_loader', fallback=False)
```

And use it directly in the if on line 82 (i.e. don't store this in an variable)

Reply:
Fixed.